test/slack-send-test.js

```diff
@@ -56,16 +56,19 @@ describe('slack-send', () => {
     describe('happy path', () => {
       it('should send a message using the postMessage API', async () => {
         fakeCore.getInput.withArgs('slack-message').returns('who let the dogs out?');
-        fakeCore.getInput.withArgs('channel-id').returns('C123456');
+        fakeCore.getInput.withArgs('channel-id').returns('C123456,C987654');
```

Comment:
Thanks for adding the test! Could we actually split this test case up into two test cases? Ideally I'd like separate tests for the single-channel and multi-channel scenarios.